exchanges/stream/buffer/buffer_test.go

```diff
@@ -12,12 +12,12 @@ import (
 )
 
 var itemArray = [][]orderbook.Item{
-	{{Price: 1000, Amount: 1, ID: 1}},
-	{{Price: 2000, Amount: 1, ID: 2}},
-	{{Price: 3000, Amount: 1, ID: 3}},
-	{{Price: 3000, Amount: 2, ID: 4}},
-	{{Price: 4000, Amount: 0, ID: 6}},
-	{{Price: 5000, Amount: 1, ID: 5}},
+	{{Price: 1000, Amount: 1, ID: 1000}},
+	{{Price: 2000, Amount: 1, ID: 2000}},
+	{{Price: 3000, Amount: 1, ID: 3000}},
+	{{Price: 3000, Amount: 2, ID: 4000}},
+	{{Price: 4000, Amount: 5, ID: 6000}},
```

Comment:
We no longer have a test which covers deleting where amount = 0 which is/was standard to declare deleting an element in the orderbook. Deletion via amount does not have test coverage anymore